src/components/KioskLayout.tsx

```
@@ -0,0 +1,801 @@
+                      </div>
+                      <div className={styles.cartItemPrice}>
+                        S/ {(item.price * item.quantity).toFixed(2)}
+                      </div>
+                    </div>
+                    <div className={styles.cartItemControls}>
+                      <div className={styles.quantityControls}>
+                        <button 
+                          className={styles.quantityButton}
+                          onClick={() => decreaseQuantity(validId, item.productType)}
+                        >
+                          <Minus size={12} />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)